java/src/main/c++/vw_learner_VWFloatArrayLearner.cc

```diff
@@ -3,9 +3,9 @@
 #include "jni_base_learner.h"
 #include "vw_learner_VWFloatArrayLearner.h"
 
-jfloatArray floatArrayPredictor(example* vec, JNIEnv *env) {
+jfloatArray floatArrayPredictor(example* vec, JNIEnv *env, vw *vwPtr) {
```

Comment:
Since we're calling the variable `vwInstance` in `jni_base_learner.h` can we rename the variable in all of these functions to be `vwInstance`.  Consistency makes this easier to follow.